sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp

```diff
@@ -1141,7 +1141,7 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
                     std::to_string(setPropertiesOptions.Timeout.GetValue())));
           }
           request.AddHeader(Details::HeaderVersion, setPropertiesOptions.ApiVersionParameter);
-          return SetPropertiesParseResult(context, pipeline.Send(context, request));
+          return SetPropertiesParseResult(context, pipeline.Send(request, context));
```

Comment:
FYI @Jinming-Hu, I left all these "Result" methods untouched across all of storage, but they all take context as the first parameter. I only focused on common public types from core.

Are they meant to be public?

Consider changing the auto-gen to move the context parameter to be last.

For example, we may want to have it be as follows:
```C++
return SetPropertiesParseResult(pipeline.Send(request, context), context);
```

Reply:
@ahsonkhan , I believe they're not meant to be public. But auto-gen of file and datalake services are maintained by @katmsft